components/hab/install.sh

```diff
@@ -29,9 +29,11 @@ main() {
   channel="stable"
   # Set an empty version variable, signaling we want the latest release
   version=""
+  # Set an empty target variable, signaling we want our current "${arch}-${sys}"
+  target=""
```

Comment:
Instead of signaling through an empty string, could we just set this to `${arch}-${sys}` up front, let the user override with `-t` if they so desire, and then have what is now `get_target` be a pure validation function?